Pod/Classes/Optional Data Sources/VOKDefaultPagingAccessory.m

```diff
@@ -40,9 +38,77 @@ - (void)didMoveToSuperview
         self.indicator = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:indicatorStyle];
         [self.indicator setTintColor:[UIColor blackColor]];
         [self.indicator setHidesWhenStopped:NO];
+        self.indicator.translatesAutoresizingMaskIntoConstraints = NO;
         
-        [self.indicator setCenter:CGPointMake(label.frame.origin.x - self.indicator.frame.size.width, self.frame.size.height/2)];
-        [self addSubview:self.indicator];
+        // Wrap the label and activity indicator in a parent view to be collectively centered
+        UIView *containerView = [[UIView alloc] init];
+        containerView.translatesAutoresizingMaskIntoConstraints = NO;
+        [containerView addSubview:self.indicator];
+        [containerView addSubview:label];
+        [self addSubview:containerView];
+        
+        // Position the activity indicator and label
+        [containerView addConstraint:[NSLayoutConstraint constraintWithItem:containerView
+                                                         attribute:NSLayoutAttributeLeading
+                                                         relatedBy:NSLayoutRelationEqual
+                                                            toItem:self.indicator
+                                                         attribute:NSLayoutAttributeLeading
+                                                        multiplier:1
+                                                          constant:0]];
+        [containerView addConstraint:[NSLayoutConstraint constraintWithItem:containerView
+                                                                  attribute:NSLayoutAttributeTrailing
+                                                                  relatedBy:NSLayoutRelationEqual
+                                                                     toItem:label
+                                                                  attribute:NSLayoutAttributeTrailing
+                                                                 multiplier:1
+                                                                   constant:0]];
+        // Put a little space between the lable and activity indicator
+        [containerView addConstraint:[NSLayoutConstraint constraintWithItem:label
+                                                                  attribute:NSLayoutAttributeLeading
+                                                                  relatedBy:NSLayoutRelationEqual
+                                                                     toItem:self.indicator
+                                                                  attribute:NSLayoutAttributeTrailing
+                                                                 multiplier:1
+                                                                   constant:20]];
```

Comment:
I'm assuming whatever the minimum iOS version is doesn't allow for using the layout anchors?